docs/ww84.html

```diff
@@ -0,0 +1,263 @@
+
+        #simulator {
+            position: relative;
+            height: 0;
+            padding-bottom: 117.6%;
+            overflow: hidden;
+            margin: 1.5em auto;
+            margin-bottom: 3em;
+        }
+
+        #simulator iframe {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+        }
+
+        .ui.button {
+            width: 14em;
+            padding: 1.5em;
+            margin: 2em auto;
+            background-image: linear-gradient(116deg, #df0571 0%, #1e00bd 100%);
+            color: #ffffff;
+            display: block;
+            font-size: 1.4em;
+            box-shadow: none;
+            transition: .25s;
+        }
+
+        .ui.button:focus,
+        .ui.button:hover {
+            background-image: linear-gradient(116deg, #df0571 0%, #1e00bd 100%);
+            box-shadow: 0 0 50px rgba(223, 5, 113, .75);
+            color: #ffffff;
+            filter: none;
+            transform: scale(1.05);
+        }
+
+        ol li::before {
+            color: #ffffff !important;
+            font-size: 2em;
+            font-weight: 300;
+            border-radius: 50%;
+
+            width: 2.5rem;
+            height: 2.5rem;
+            margin-left: -3rem !important;
+            margin-top: -0.5rem !important;
+
+            padding-right: 0.7rem;
+            padding-top: 0.6rem;
+
+            background-color: #df0571 !important;
+            ;
+        }
+
+        ol.ui.list {
+            padding: 0 2em;
+            margin: 1em 1.25em;
+            align-self: stretch;
+        }
+
+        ol li p {
+            padding-left: 0.5em;
+            line-height: 1.5em;
+        }
+
+        .cta {
+            text-align: center;
+            max-width: 550px;
+            padding-bottom: 2em;
+        }
+
+        .logotext {
+            max-width: 30%;
+        }
+
+        /** footer styling **/
+        footer {
+            background-color: #0d0015 !important;
+            border-top: 1px solid hsla(0, 0%, 100%, 0.2) !important;
+            margin-top: 0 !important;
+            padding-top: 2rem !important;
+        }
+
+        footer.ui * {
+            color: white !important;
+            opacity: 0.9;
+        }
+
+        .ui.divider:not(.vertical):not(.horizontal) {
+            border-top: 1px solid hsla(0, 0%, 100%, 0.3);
+            border-bottom: 1px solid hsla(0, 0%, 55%, 0.1);
+        }
+
+        @media only screen and (max-width: 813px) {
+            #main {
+                margin: 0;
+                padding: 1em;
+            }
+
+            .header-image {
+                margin: -1em;
+            }
+
+            ol.ui.list {
+                margin: 0 0;
+            }
+
+            h1 {
+                font-size: 2.3em;
+            }
+        }
+    </style>
+
+    <script>
+        function registerTickEvents() {
+            function tickEvent(id, data, measures) {
```

Comment:
small: would be nice to start using tickevent.html to unify our telemetry on these static pages

Reply:
Ahh, forgot about that; pushed a quick change to use that & also remove the langpicker element that's not supported on these side pages anyways